src/Yoti/Entity/Anchor.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Yoti\Entity;
+
+
+class Anchor
+{
+    /**
+     * @var string
+     */
+    protected $value;
+
+    /**
+     * @var string
+     */
+    protected $subType;
+
+    /**
+     * @var \Protobuf\Stream
+     */
+    protected $signature;
+
+    protected $artifactSignature;
+
+    /**
+     * @var \Protobuf\Stream
+     */
+    protected $signedTimeStamp;
+
+    /**
+     * @var int
+     */
+    protected $timeStamp;
+
+    /**
+     * @var array
+     */
+    protected $originServerCerts;
+
+    public function __construct(
+        $value,
+        $subType,
+        $signature,
+        $artifactSignature,
+        $signedTimeStamp,
+        array $originServerCerts
+    ) {
+        $this->value = $value;
+        $this->subType = $subType;
+        $this->signature = $signature;
+        $this->artifactSignature = $artifactSignature;
+        $this->signedTimeStamp = $signedTimeStamp;
+        $this->originServerCerts = $originServerCerts;
+
+        $config = new \Protobuf\Configuration();
+        $streamReader = $config->getStreamReader();
+        $this->timeStamp = $streamReader->readFixed32($this->signedTimeStamp);
```

Comment:
Once you have converted the stream to an object as explained above, you can delete this as that object should have a `getTimestamp` method itself.